src/plugins/intel_gpu/include/intel_gpu/runtime/format.hpp

```diff
@@ -380,46 +387,48 @@ struct format {
                               bool is_nv12 = false);
 
     /// @brief Checks if @p format is of grouped type
-    static bool is_grouped(type fmt) { return group_num(fmt) != 0; }
+    static bool is_grouped(const format& fmt) { return group_num(fmt) != 0; }
     /// @brief Checks if @p format is of image type
-    static bool is_image(type fmt) { return (is_image_2d(fmt)); }
+    static bool is_image(const format& fmt) { return (is_image_2d(fmt)); }
     /// @brief Checks if @p format is blocked format
-    static bool is_blocked(type fmt) { return !(block_sizes(fmt).empty()); }
+    static bool is_blocked(const format& fmt) { return !(block_sizes(fmt).empty()); }
     /// @brief Checks if @p format is blocked format which has single inner block
-    static bool is_single_blocked(type fmt) { return (block_sizes(fmt).size() == 1); }
+    static bool is_single_blocked(const format& fmt) { return (block_sizes(fmt).size() == 1); }
     /// @brief Checks if @p format is blocked format which has multiple inner blocks
-    static bool is_multi_blocked(type fmt) { return (block_sizes(fmt).size() > 1); }
+    static bool is_multi_blocked(const format& fmt) { return (block_sizes(fmt).size() > 1); }
     /// @brief Checks if @p format is nv12 format
-    static bool is_nv12(type fmt) { return (fmt == nv12); }
+    static bool is_nv12(const format& fmt) { return (fmt == nv12); }
 
     /// @brief Returns number of batch dimensions.
-    size_t batch_num() const { return traits(value).batch_num; }
+    size_t batch_num() const { return traits().batch_num; }
     /// @brief Returns number of feature dimensions.
-    size_t feature_num() const { return traits(value).feature_num; }
+    size_t feature_num() const { return traits().feature_num; }
     /// @brief Returns number of spatial dimensions.
-    size_t spatial_num() const { return traits(value).spatial_num; }
+    size_t spatial_num() const { return traits().spatial_num; }
     /// @brief Returns number of group dimensions.
-    size_t group_num() const { return traits(value).group_num; }
+    size_t group_num() const { return traits().group_num; }
     /// @brief Returns an order of dimensions.
-    const std::vector<uint64_t>& dims_order() const { return traits(value)._order; }
+    const std::vector<uint64_t>& dims_order() const { return traits()._order; }
     /// @brief Returns an order of dimensions in form of string.
-    const std::string& order() const { return traits(value).order; }
+    const std::string& order() const { return traits().order; }
     /// @brief Returns an internal orders of dimensions form of string.
-    const std::string& internal_order() const { return traits(value).internal_order; }
+    const std::string& internal_order() const { return traits().internal_order; }
+    /// @brief Returns an order of dimensions for a @ format.
+    const std::vector<size_t>& axes_order() const { return traits()._order; }
```

Comment:
This is a duplication of `const std::vector<uint64_t>& dims_order() const { return traits()._order; }`

Reply:
Removed